presto-accumulo/src/test/java/com/facebook/presto/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -0,0 +1,503 @@
+//                "SELECT 10");
+
+        // Tests for CREATE TABLE with UNION ALL: exercises PushTableWriteThroughUnion optimizer
+
+//        Function "UUID" not found
+//        this.assertCreateTableAsSelect(
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders WHERE orderkey % 2 = 0 UNION ALL " +
+//                        "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders WHERE orderkey % 2 = 1",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders",
+//                "SELECT count(*) FROM orders");
+
+//        Function "UUID" not found
+//        this.assertCreateTableAsSelect(
+//                getSession().withSystemProperty("redistribute_writes", "true"),
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT count(*) + 1 FROM orders");
+
+//        Function "UUID" not found
+//        this.assertCreateTableAsSelect(
+//                getSession().withSystemProperty("redistribute_writes", "false"),
+//                "test_union_all",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT UUID() AS uuid, orderdate, orderkey, totalprice FROM orders UNION ALL " +
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT count(*) + 1 FROM orders");
+    }
+
+    @Override
+    public void testDelete()
+            throws Exception
+    {
+        try {
+            // TODO Deletes are not supported by the connector
+            super.testDelete();
+        }
+        catch (Exception e) {
+            assertUpdate("DROP TABLE test_delete");
+            assertEquals("This connector does not support updates or deletes", e.getMessage());
+        }
+    }
+
+    @Override
+    public void testInsert()
+            throws Exception
+    {
+        @Language("SQL") String query = "SELECT UUID() AS uuid, orderdate, orderkey FROM orders";
+
+        assertUpdate("CREATE TABLE test_insert AS " + query + " WITH NO DATA", 0);
+        assertQuery("SELECT count(*) FROM test_insert", "SELECT 0");
+
+        assertUpdate("INSERT INTO test_insert " + query, "SELECT count(*) FROM orders");
+
+        assertQuery("SELECT orderdate, orderkey FROM test_insert", "SELECT orderdate, orderkey FROM orders");
+        // Override because base class error: Cannot insert null row ID
```

Comment:
Why? Nulls are not supported by Accumulo? Then how you achieve this while adding column using https://github.com/bloomberg/presto-accumulo/tree/master/presto-accumulo-tools? There are nulls for added columns.